codecov_auth/migrations/0034_oktauser.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 4.2.2 on 2023-07-25 18:08
+
+import uuid
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("codecov_auth", "0033_sentryuser"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="OktaUser",
+            fields=[
+                ("id", models.BigAutoField(primary_key=True, serialize=False)),
+                ("external_id", models.UUIDField(default=uuid.uuid4, editable=False)),
+                ("created_at", models.DateTimeField(auto_now_add=True)),
```

Comment:
Hmmm, when we use the `BaseCodecovModel`, we don't save this with the `...WithoutTZField`, we okay with that?

Reply:
I think it's probably fine.  We're extending from `BaseCodecovModel` in a lot of other models as well.